src/docdb/tree/DocDBDocumentsTreeItem.ts

```diff
@@ -0,0 +1,78 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+import * as vscode from 'vscode';
+import { DocumentClient, QueryIterator, FeedOptions, RetrievedDocument } from 'documentdb';
+import { DocDBTreeItemBase } from './DocDBTreeItemBase';
+import { IAzureTreeItem, UserCancelledError, IAzureNode } from 'vscode-azureextensionui';
+import { DocDBDocumentTreeItem } from './DocDBDocumentTreeItem';
+import { DocDBCollectionTreeItem } from './DocDBCollectionTreeItem';
+
+/**
+ * This class provides logic for DocumentDB collections
+ */
+export class DocDBDocumentsTreeItem extends DocDBTreeItemBase<RetrievedDocument> {
+    public static contextValue: string = "cosmosDBDocumentsGroup";
+    public readonly contextValue: string = DocDBDocumentsTreeItem.contextValue;
+    public readonly childTypeLabel: string = "Documents";
+
+    constructor(documentEndpoint: string, masterKey: string, private _collection: DocDBCollectionTreeItem, isEmulator: boolean) {
+        super(documentEndpoint, masterKey, isEmulator);
+    }
+
+    public get iconPath(): any {
+        return {
+            light: path.join(__filename, '..', '..', '..', '..', '..', 'resources', 'icons', 'theme-agnostic', 'Collection.svg'),
+            dark: path.join(__filename, '..', '..', '..', '..', '..', 'resources', 'icons', 'theme-agnostic', 'Collection.svg')
+        };
+    }
+
+    public get id(): string {
+        return "$Documents";
```

Comment:
nit: Remember id is an optional field - you don't have to specify it in in a case like this where the label should already be unique at this level of the tree

Reply:
Abstract in the inherited code, not worth changing.